apps/storybook/package.json

```diff
@@ -28,7 +28,7 @@
     "react-dom": "^18.0.0",
     "react-table": "^7.1.0",
     "serve": "^14.0.1",
-    "storybook": "~7.0.2",
+    "storybook": "~0.0.0",
```

Comment:
note for future: our `storybook` workspace conflicts with the `storybook` npm package. This is causing `changesets` to make this change. Will need to investigate how to fix this